src/code/stores/graph-store.ts

```diff
@@ -965,15 +965,16 @@ export const GraphStore: GraphStoreClass = Reflux.createStore({
     })();
     const settings = AppSettingsStore.serialize();
     settings.simulation = SimulationStore.serialize();
-    const topology = getTopology({nodes: nodeExports, links: linkExports});
+    // TODO: fix and re-enable https://www.pivotaltracker.com/n/projects/1263626/stories/170410042
+    // const topology = getTopology({nodes: nodeExports, links: linkExports});
     const data = {
       version: latestVersion(),
       filename: this.filename,
       palette,
       nodes: nodeExports,
       links: linkExports,
       settings,
```

Comment:
Trailing comma?

Reply:
The Typescript compiler rewrites all of this so it doesn't matter.